src/framework/global/utils.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "utils.h"
+
+#include "translation.h"
+
+using namespace mu;
+
+static const std::vector<std::string> noteHeadNamesLower = {
+    trc("global", "c"),
```

Comment:
I believe these static variables won't work well with translations. They are translated when they are needed the first time, but not updated when the user changes the app language. Instead you would need to use QT_TRANSLATE_NOOP and then call trc or qtrc every time the string is needed.

Reply:
thanks, fixed